README.md

```diff
@@ -51,13 +51,24 @@ rather than IP numbers in a Hyper-V environment. On CentOS systems,
 this requires the bind-utils package (updatedns requires the 'dig'
 program). The bind-utils package isn't otherwise necessary.
 
+- Install minimum version of Ruby (the system must have 1.9 or higher, but 2.2.6 is recommended)
```

Comment:
The compatibility issue that was found was resolved I believe in versions starting 2. I am not sure about the minor, but I can try finding it for you.